src/shared/components/annotation/CancerHotspots.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import * as _ from 'lodash';
-import Tooltip from 'rc-tooltip';
+import DefaultTooltip from 'shared/components/DefaultTooltip';
```

Comment:
It works, as in, it compiles, but the IDE won't give hints to the path, which I find to be very annoying.